src/predefinedComponents/AvatarHeader/AvatarHeader.tsx

```diff
@@ -248,93 +259,58 @@ class AvatarHeader extends React.Component {
       <>
         <StatusBar backgroundColor={backgroundColor} barStyle="light-content" />
         <StickyParallaxHeader
-          contentContainerStyles={contentContainerStyles}
-          foreground={this.renderForeground()}
-          header={this.renderHeader()}
-          deviceWidth={constants.deviceWidth}
           scrollEvent={event([{ nativeEvent: { contentOffset: { y: this.scrollY.y } } }], {
             useNativeDriver: false,
-            listener: (e) => scrollEvent && scrollEvent(e),
+            listener: (e: NativeSyntheticEvent<NativeScrollEvent>) => scrollEvent && scrollEvent(e),
           })}
-          headerSize={this.setHeaderSize}
-          headerHeight={headerHeight}
+          // deviceWidth={constants.deviceWidth}
```

Comment:
Should we remove it? Maybe it was a regression where prop was not passed at all?

Reply:
`deviceWidth` is not used anywhere in code, while refactoring using typescript i didn't found any usage of `deviceWidth` and in all components prop was missing in prop types.